src/backend/access/appendonly/appendonly_compaction.c

```diff
@@ -519,8 +520,10 @@ AppendOnlyCollectDeadSegments(Relation aorel, List *compaction_segno)
 	Assert(Gp_role == GP_ROLE_EXECUTE || Gp_role == GP_ROLE_UTILITY);
 	Assert(RelationIsAoRows(aorel));
 
+	relname = RelationGetRelationName(aorel);
```

Comment:
Does this produce a compilation warning without explicitly casting `char *` to `const char *`?

Reply:
I did as [in](https://github.com/arenadata/gpdb/pull/695/files#diff-4d241dcb0deba3493d7dfa8c8488e87e032194e003373bf9b48bfe8685c6f4faR615-R633)